star_tides/api/util/issue_jwt.py

```diff
@@ -5,16 +5,15 @@
 from flask import current_app
 
 
-def create_jwt(email):
+def create_jwt(user_id):
     now = datetime.now(timezone.utc)
-
     jwt_token = jwt.encode(
         {
             'iss': 'star-tides',
             'exp': now + timedelta(hours=1),
             'iat': now,
             'claims': {
-                'email': email
+                'id': str(user_id)
```

Comment:
Why not have both the email and the ID in the claims? Does it complicate (or simplify) any code to have both?

Reply:
Because it'll be passed on every single network request, I want to keep this lean